### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jkotas**
- Delete bit-rotten netstandard13aot and netstandard15aot support that never shipped
- Delete files moved to CoreLib that no longer build

### CODE REVIEW

**Comment by jkotas** [comment]
Pealing off independent cleanup from https://github.com/dotnet/corefx/pull/12187

cc @stephentoub @weshaggard

File: src/System.IO/pkg/aot/System.IO.pkgproj [hunk 1]
```diff
@@ -7,9 +7,6 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\System.IO.csproj">
-      <TargetGroup>netstandard13aot</TargetGroup>
```
Comment (weshaggard) [inline src/System.IO/pkg/aot/System.IO.pkgproj]: What is your expectation for libraries that need to target CoreRT? What target group do you expect them to build with?

Comment (weshaggard) [inline src/System.IO/pkg/aot/System.IO.pkgproj]: Should we create a netcoreapp12aot TFM for them?

Comment (jkotas) [inline src/System.IO/pkg/aot/System.IO.pkgproj]: Yes, something like that ... @joshfree or @SedarG may have more detailed idea about the plan.

**Comment by stephentoub** [comment]
@dotnet-bot Test Innerloop Ubuntu14.04 Release Build and Test please (build hang)
FYI, @adityamandaleeka, @mmitche

**Pull request merged**